src/pages/certificates/hooks/use-certificates.js

```diff
@@ -37,9 +37,9 @@ const transformDate = (date) => {
 const checkStatus = (certificate) => {
```

Comment:
Will something be returned from this function?

Reply:
it will return the string